.travis.yml

```diff
@@ -0,0 +1,69 @@
+sudo: false
```

Comment:
not sure why we have a travis.yml, don't you need a 3rd party subscription for this, if not please educate me.